modules/gdscript/language_server/gdscript_workspace.cpp

```diff
@@ -279,6 +279,8 @@ ExtendGDScriptParser *GDScriptWorkspace::get_parse_result(const String &p_path)
 	return nullptr;
 }
 
+#define HANDLE_DOC(m_string) ((is_native ? DTR(m_string) : (m_string)).strip_edges())
```

Comment:
I'm late, but you forgot to add `#undef HANDLE_DOC` at the end. This is not required, but is recommended for good style and to prevent potential conflicts in SCU builds.